common/arduino/variant.c

```diff
@@ -24,6 +24,8 @@
 #define PRINT           printk
 #endif
 
+#define PINMUX_NAME CONFIG_PINMUX_NAME
```

Comment:
Actually, I take my approval back.... what purpose is this define serving?

We already have "CONFIG_PINMUX_NAME", which everyone will understand instantly (it's a config file option).

Re-defining it without the CONFIG_ part is just confusing.